misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/manufacturing/ReceiptReceiveTargetScreen.jsx

```diff
@@ -17,9 +18,9 @@ class ReceiptReceiveTargetScreen extends PureComponent {
   };
 
   handleScanClick = () => {
-    const { wfProcessId, activityId, lineId, dispatch, appId } = this.props;
+    const { dispatch } = this.props;
     const stepId = 'receipt';
-    const location = `/workflow/${wfProcessId}/activityId/${activityId}/lineId/${lineId}/stepId/${stepId}/scanner/${appId}`;
+    const location = getLocation({ ...this.props, stepId }, true);
```

Comment:
is it necessary to pass all the props in here ? wouldn't be more clear if we pass only the explicit values ?

Reply:
pls use a specific location function.

NOTE: in future we will decouple these apps more and more because they will grow totally independent.